src/pretix/control/navigation.py

```diff
@@ -343,10 +343,24 @@ def get_global_navigation(request):
             'icon': 'group',
         },
         {
-            'label': _('Order search'),
+            'label': _('Search'),
             'url': reverse('control:search.orders'),
-            'active': 'search.orders' in url.url_name,
+            'active': False,
             'icon': 'search',
+            'children': [
+                {
+                    'label': _('Order search'),
+                    'url': reverse('control:search.orders'),
+                    'active': 'search.orders' in url.url_name,
+                    'icon': 'search',
+                },
+                {
+                    'label': _('Payment search'),
```

Comment:
I'm wondering, if we already call the menu section "Search", wouldn't "Orders" and "Payments" be a sufficient description of the sub-items? It feels a little bit doubled to repeat the "search" part